samples/drivers/led_ws2812/boards/esp32c3_devkitm.overlay

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2022 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/led/led.h>
+
+&spi2 {
```

Comment:
@marekmatej, can you add `line-idle-low;` in all `spi`entries after #60183?

Reply:
I needed to add this to make it work for the esp32c3 I tested.